bindings/rust/bench/README.md

```diff
@@ -0,0 +1,43 @@
+# Benchmarking s2n-tls
+
+We benchmark s2n-tls against two other common TLS libraries, Rustls and OpenSSL. We use Rust bindings for s2n-tls and OpenSSL and Criterion to benchmark these libraries. 
+
+## Setup 
+
+Setup is easy! Just run `/bindings/rust/generate.sh` to generate Rust bindings for s2n-tls and have OpenSSL installed.
+
+## Running benchmarks
+
+To run our results, just run `cargo bench`! Critierion will auto-generate an HTML report in `target/criterion/`. 
+
+## Future features
```

Comment:
I'm not sure about this paragraph. I don't know if it's that useful for readers to know what we're doing in the future, given that we have no timeline for any of this stuff. I would take it out unless you think it's important.

Reply:
These all should be done in the next few weeks, and I have PRs lined up for the first three, so I think it's reasonable to keep them. The only exception is session resumption, which we decided to be outside of the scope of this project until session resumption Rust bindings for s2n-tls are exposed.